pkg/reconciler/knativeserving/ingress/ingress_test.go

```diff
@@ -78,8 +77,9 @@ func TestAppendInstalledIngresses(t *testing.T) {
 	tests := []struct {
 		name                 string
 		instance             servingv1alpha1.KnativeServing
-		expected             bool
+		expectedNoErr        bool
```

Comment:
Do we need this `expectedNoErr`? The test cases always expect `true`.

Reply:
I will change the `expectError` into a real error, instead of the bool.
If it is nil, check the equality. if it is an error, check the string of the error.